ui/src/ducks/app/alerts.js

```diff
@@ -35,7 +42,7 @@ export const fetchAlertsAlertmanagerAction = () => {
   return { type: FETCH_ALERTS_ALERTMANAGER };
 };
 
-export const updateAlertsAlertmanagerAction = (payload) => {
+export const updateAlertsAlertmanagerAction = (payload: $Shape<AlertsState>) => {
```

Comment:
Oh, neat type 😄 , I guess we'll be able to use it for many "update" action creators.